examples/build.sbt

```diff
@@ -6,47 +6,63 @@ benchmarks ++= Seq(
   suiteForEncoding_examples(examplesSpecs)
 )
 
+// (folder, spec, init, inv, length)
 lazy val examplesSpecs = Seq(
-  ("MC4_FALSE_FALSE.tla", "Consistency"),
-  ("MC4_TRUE_TRUE.tla", "Consistency"),
-  ("MC10_FALSE_FALSE.tla", "Consistency"),
-  ("MC10_TRUE_TRUE.tla", "Consistency"),
-  ("MC20_FALSE_FALSE.tla", "Consistency"),
-  ("MC20_TRUE_TRUE.tla", "Consistency")
+  ("2PCwithBTM", "MC4_FALSE_FALSE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC4_TRUE_TRUE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC10_FALSE_FALSE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC10_TRUE_TRUE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC20_FALSE_FALSE.tla", "Init", "Consistency", "10"),
+  ("2PCwithBTM", "MC20_TRUE_TRUE.tla", "Init", "Consistency", "10"),
+  ("aba_asyn_byz", "MC4.tla", "Init", "", "10"),
+  ("aba_asyn_byz", "MC10.tla", "Init", "", "10"),
+  ("aba_asyn_byz", "MC20.tla", "Init", "", "10"),
+  ("bakery", "MC3.tla", "Init", "MutualExclusion", "10"),
+  ("bakery", "MC3.tla", "Init", "Inv", "0"),
+  ("bakery", "MC3.tla", "Inv", "Inv", "1"),
+  ("changRoberts", "MC4.tla", "Init", "Correctness", "10"),
+  ("changRoberts", "MC10.tla", "Init", "Correctness", "10"),
+  ("changRoberts", "MC20.tla", "Init", "Correctness", "10"),
+  ("paxos", "MC3.tla", "Init", "V!OneValuePerBallot", "13"),
+  ("readersWriters", "MC4.tla", "Init", "Safety", "10"),
+  ("readersWriters", "MC10.tla", "Init", "Safety", "10"),
+  ("readersWriters", "MC20.tla", "Init", "Safety", "10")
 )
 
-def suiteForEncoding_examples(specs: Seq[(String, String)]) = {
+def suiteForEncoding_examples(specs: Seq[(String, String, String, String, String)]) = {
   val suiteTimeLimit = "1h"
 
-  def checkCmd(encoding: String, inv: String, searchInvMode: String, discardDisabled: String) = {
+  def checkCmd(init: String, inv: String, length: String, encoding: String, searchInvMode: String, discardDisabled: String) = {
```

Comment:
imo, 6 args is also too many (generally) for a function. Using a case class to package the fields as a record will save having to shuffle params around and clean this up.